pkg/runner/test_runner.go

```diff
@@ -532,7 +534,7 @@ func (tr *TestRunner) WaitPipelineTermination(ch completionCh, bundles []test.Te
 // have completed or an error occurs. If all Targets complete successfully, it checks
 // whether TestSteps and routing blocks have completed as well. If not, returns an
 // error. Termination is signalled via terminate channel.
-func (tr *TestRunner) WaitPipelineCompletion(terminate <-chan struct{}, ch completionCh, bundles []test.TestStepBundle, targets []*target.Target) error {
+func (tr *TestRunner) WaitPipelineCompletion(terminate <-chan struct{}, ch completionCh, t *test.Test, bundles []test.TestStepBundle, targets []*target.Target) error {
```

Comment:
`test.Test` is necessary to get the test name in this function to emit a fully populated event